pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -687,6 +687,14 @@ func GetRecordingRules(namespace string) []KubevirtRecordingRule {
 			MType:       prometheusv1.MetricTypeCounter,
 			Description: "The total number of requests to deprecated KubeVirt APIs.",
 		},
+		{
+			Rule: v1.Rule{
+				Record: "kubevirt_migrate_vmi_disk_transfer_rate_bytes",
+				Expr:   intstr.FromString("irate(kubevirt_migrate_vmi_data_processed_bytes[5m])"),
```

Comment:
@machadovilaca do we see a separation between the memory and the disk during the migration?
I was thinking that total processed data is memory + disk, isn't it?

Reply:
I think this will not be significant to migrations, but rather to block jobs in general... 
Perhaps this disk info is just no longer available. 
I would say that we should just drop this metric and re-add once we have a better solution.